packages/engine-ui/src/components/Sidebar.tsx

```diff
@@ -19,9 +20,21 @@ const Container = styled.div`
 const Content = styled.div`
   padding: 2rem;
   width: 350px;
+  overflow: auto;
 `
 
-const Sidebar = (state) => {
+interface props {
```

Comment:
Should this be capitalized, for convention?

Reply:
Yes, will fix!